lib/game.rb

```diff
@@ -0,0 +1,14 @@
+class Game
+attr_reader :name, :cost
+
+  def initialize(name, cost, nat_draw = false)
+    @name = name
+    @cost = cost
+    @nat_draw = nat_draw
```

Comment:
try not to abbreviate-- someone looking at this a year from now with no context may have absolutely no idea what `nat_draw` is supposed to mean.